console.log('[SyntaxError: Unexpected end of input]')
try {} catch (e) {}
